guides/notebooks/stylesheets.livemd

```diff
@@ -0,0 +1,456 @@
+
+Generally, using RGB values allows for quicker iteration, while using the asset catalog is more performant and customizable.
+
+<!-- livebook:{"break_markdown":true} -->
+
+### Custom Colors in Stylesheets
+
+Here's an example of defining a custom color. We'll apply this `custom-color` class to a SwiftUI Text View in a moment.
+
+```elixir
+defmodule CustomColorStylesheet do
+  use LiveViewNative.Stylesheet, :swiftui
+
+  ~SHEET"""
+  "custom-color" do
+    # light blue
+    foregroundStyle(Color(.sRGB, red: 0.4627, green: 0.8392, blue: 1.0))
+  end
+  """
+end
+```
+
+Evaluate the example below and view the custom color in your simulator.
+
+<!-- livebook:{"attrs":"eyJhY3Rpb24iOiI6aW5kZXgiLCJjb2RlIjoiZGVmbW9kdWxlIFNlcnZlci5DdXN0b21Db2xvclNoZWV0TGl2ZSBkb1xuICB1c2UgUGhvZW5peC5MaXZlVmlld1xuICB1c2UgTGl2ZVZpZXdOYXRpdmUuTGl2ZVZpZXdcbiAgdXNlIEN1c3RvbUNvbG9yU3R5bGVzaGVldFxuXG4gIEBpbXBsIHRydWVcbiAgZGVmIHJlbmRlcigle2Zvcm1hdDogOnN3aWZ0dWl9ID0gYXNzaWducykgZG9cbiAgICB+U1dJRlRVSVwiXCJcIlxuICAgIDxUZXh0IGNsYXNzPVwiY3VzdG9tLWNvbG9yXCI+SGVsbG8gZnJvbSBMaXZlVmlldyBOYXRpdmUhPC9UZXh0PlxuICAgIFwiXCJcIlxuICBlbmRcbmVuZCIsInBhdGgiOiIvIn0","chunks":[[0,109],[111,282],[395,45],[442,63]],"kind":"Elixir.KinoLiveViewNative","livebook_object":"smart_cell"} -->
+
+```elixir
+require KinoLiveViewNative.Livebook
+import KinoLiveViewNative.Livebook
+import Kernel, except: [defmodule: 2]
+
+defmodule Server.CustomColorSheetLive do
+  use Phoenix.LiveView
+  use LiveViewNative.LiveView
+  use CustomColorStylesheet
+
+  @impl true
+  def render(%{format: :swiftui} = assigns) do
+    ~SWIFTUI"""
+    <Text class="custom-color">Hello from LiveView Native!</Text>
+    """
+  end
+end
+|> KinoLiveViewNative.register("/", ":index")
+
+import KinoLiveViewNative.Livebook, only: []
+import Kernel
+:ok
+```
+
+### Your Turn: Custom Colors in the Asset Catalog
+
+Custom colors can be defined in the asset catalog (https://developer.apple.com/documentation/xcode/managing-assets-with-asset-catalogs). You're going to define a new color in your iOS App.
+
+To create a new image go to the `Assets` folder in your iOS app and create a new color set.
+
+<!-- livebook:{"break_markdown":true} -->
+
+![](https://github.com/liveview-native/documentation_assets/blob/main/asset-catalogue-create-new-color-set.png?raw=true)
+
+<!-- livebook:{"break_markdown":true} -->
+
+To create a color set, enter the RGB values or a hexcode.
+
+<!-- livebook:{"break_markdown":true} -->
+
+![](https://github.com/liveview-native/documentation_assets/blob/main/asset-catalogue-modify-my-color.png?raw=true)
+
+<!-- livebook:{"break_markdown":true} -->
+
+The defined color is now available for use within Stylesheets. Evaluate the code below. You'll use the `my-color` class in a moment.
```

Comment:
The app needs to be recompiled after adding a color set. This could be noted after the instructions on adding a color.